docker/run.sh

```diff
@@ -1,11 +1,26 @@
 #!/usr/bin/env bash
 set -e
 
-# This script is provided for backwards compatibility with the old docker setup.
-# it doesn't do much aside from wrapping the usual docker compose CLI.
+run() {
+  local scriptdir=$(dirname "${BASH_SOURCE[0]}")
+  cd "$scriptdir" || exit 1
 
-SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
-cd "$SCRIPTDIR" || exit 1
+  local build_args=""
+  local service_name="invokeai-cpu"
 
-docker compose up -d
-docker compose logs -f
+  [[ -f ".env" ]] &&
+    build_args=$(awk '$1 ~ /=[^$]/ {print "--build-arg " $0 " "}' .env) &&
```

Comment:
This should avoid commented lines in `.env` - prevents `--build-arg #SOME_VAR=foo`

```suggestion
    build_args=$(awk '$1 ~ /=[^$]/ && $0 !~ /^#/ {print "--build-arg " $0 " "}' .env) &&
```